luigi/contrib/external_program.py

```diff
@@ -118,6 +118,9 @@ def _clean_output_file(self, file_object):
         file_object.seek(0)
         return ''.join(map(lambda s: s.decode('utf-8'), file_object.readlines()))
 
+    def build_tracking_url(self, logs_output):
```

Comment:
Sorry for going back and forth. Adding docstring here would be very helpful for others to understand the need of this method.

Reply:
@honnix done